presto-main/src/test/java/com/facebook/presto/sql/query/TestSubqueries.java

```diff
@@ -86,16 +86,16 @@ public void testCorrelatedExistsSubqueriesWithOrPredicateAndNull()
     }
 
     @Test
-    public void testUnsupportedSubqueriesWithCoercions()
+    public void testSubqueriesWithCoercions()
     {
         // coercion from subquery symbol type to correlation type
-        assertions.assertFails(
-                "select (select count(*) from (values 1) t(a) where t.a=t2.b limit 1) from (values 1.0) t2(b)",
-                UNSUPPORTED_CORRELATED_SUBQUERY_ERROR_MSG);
+        assertions.assertQuery(
+                "select (select count(*) from (values 1) t(a) where t.a=t2.b group by a limit 1) from (values 1.0) t2(b)",
```

Comment:
why does this need a group by? or limit for that matter

Reply:
this was from original sql statement, but on second, we don't need it since the result is a count aggregation.